mycore-base/src/main/java/org/mycore/util/concurrent/MCRFixedUserCallable.java

```diff
@@ -21,55 +21,40 @@
 import java.util.Objects;
 import java.util.concurrent.Callable;
 
-import org.mycore.common.MCRException;
 import org.mycore.common.MCRSession;
-import org.mycore.common.MCRSessionMgr;
+import org.mycore.common.MCRUsageException;
 import org.mycore.common.MCRUserInformation;
 
 /**
  * Encapsulates a {@link Callable} with a mycore session belonging to a specific user and a database transaction.
- * 
- * @author Matthias Eichner
```

Comment:
Why remove author tags? Think you should rather add yourself there too.

Reply:
Git can keep track of authorship way better then the authors themselves. The author tags are mostly wrong throughout the entire project and therefore useless.